core/backend/src/CheckpointManager.ts

```diff
@@ -48,7 +48,7 @@ export interface CheckpointProps {
 export type ProgressFunction = (loaded: number, total: number) => number;
 
 /** The parameters that specify a request to download a checkpoint file from iModelHub.
- * @beta
+ * @internal

```

Comment:
@calebmshafer I thought I would make this api public for downloading V2 checkpoints. But, the public way to do that is `BriefcaseManager.downloadBriefcase` that takes a `RequestNewBriefcaseArg`, so everything about `CheckpointManager` should be internal.